projects/packages/my-jetpack/_inc/components/add-license-screen/index.jsx

```diff
@@ -45,6 +40,14 @@ export default function AddLicenseScreen() {
 		setHasActivatedLicense( true );
 	}, [] );
 
+	const availableLicenses = useMemo(
+		() =>
+			licenses.filter(
+				( { attached_at, revoked_at } ) => attached_at === null && revoked_at === null
+			),
+		[ licenses ]
+	);
+
```

Comment:
Oh i didn't see you updated it this way, I like this better and I think it is better practices to do it this way. Thanks!